app/views/NewServerView.js

```diff
@@ -142,6 +253,44 @@ class NewServerView extends React.Component {
 		);
 	}
 
+	onChangePassword = (value) => {
+		const { certificate } = this.state;
+		this.setState({ certificate: { ...certificate, password: value } });
+	}
+
+	renderCertificatePassword = () => {
+		const { showPasswordAlert } = this.state;
+		return (
+			<Dialog.Container visible={showPasswordAlert}>
```

Comment:
This lib feels so bad :(
Can we do it using RN's AlertIOS? https://facebook.github.io/react-native/docs/alertios.html

Reply:
fixed :)
I used `Alert` because: https://github.com/facebook/react-native/issues/25744